docs/future-work/motor-system-improvements.md

```diff
@@ -5,13 +5,14 @@ description: Improvements we would like to add to the motor system.
 
 These are the things we would like to implement:
 
-- [Use a different policy for learning than for inference](motor-system-improvements/use-a-different-policy-for-learning-than-for-inference.md) #learning
+- [Use a different policy for learning than for inference](motor-system-improvements/enable-policy-switching-learning-vs-inference.md) #learning
 - [Bottom-Up exploration policy for surface agent](motor-system-improvements/bottom-up-exploration-policy-for-surface-agent.md) #learning
 - [Top-down exploration policy](motor-system-improvements/top-down-exploration-policy.md) #learning #numsteps
-- [Bottom up policies for distant agent](motor-system-improvements/bottom-up-policies-for-distant-agent.md) #numsteps
-- [Calculate saliency map in view finder and use it to saccade to region of interest ](motor-system-improvements/calculate-saliency-map-in-view-finder-and-use-it-to-saccade-to-region-of-interest.md)#numsteps #multiobj
+- [Calculate saliency map in view finder and use it to saccade to region of interest ](motor-system-improvements/implement-efficient-saccades-driven-by-model-free-and-model-based-signals.md)#numsteps #multiobj
 - [Learn policy using RL](motor-system-improvements/learn-policy-using-rl.md) (using simplified action space) #numsteps
 - [Decompose goals into subgoals & communicate](motor-system-improvements/decompose-goals-into-subgoals-communicate.md) #goalpolicy
-- [Interprete goal states in motor system & switch policies](motor-system-improvements/interprete-goal-states-in-motor-system-switch-policies.md) #goalpolicy
+- [Interprete goal states in motor system & switch policies](motor-system-improvements/interpret-goal-states-in-motor-system-switch-policies.md) #goalpolicy
+- [Reuse hypothesis testing points](motor-system-improvements/reuse-hypothesis-testing-points.md) #goalpolicy #numsteps
+- [Implement a simple cross-modal policy](motor-system-improvements/simple-cross-modal-policy.md) #crossmodal
```

Comment:
Oh good catch, that was a Cursor hallucination. I've changed it to `#learning #multiobj #goalpolicy #numsteps` - I feel learning in the sense that we can support learning a detailed model that we might only be able to see one angle, and this could help in learning in multi-object settings. `#goalpolicy` because of the policy element, and maybe `numsteps` more than speed since it's more about convergence than computational optimization?